os-autoinst-obs-auto-submit

```diff
@@ -18,10 +18,33 @@ XMLSTARLET=$(command -v xmlstarlet || true)
 # shellcheck source=/dev/null
 . "$(dirname "${BASH_SOURCE[0]}")"/_common
 
+check_project_exists() {
+    # ex: openSUSE:Backports:SLE-15-SP6
+    local prj="$1"
+    local rc=0
+    if [[ -z "$prj" ]]; then
+        echo "Error: No project provided." >&2
+        return 1
+    fi
+    # dont need the XML output for checking existence.
+    # if $prj does not exist return 404
+    osc meta prj "$prj" > /dev/null 2> /dev/null
+    rc=$?
+    return "$rc"
+}
+
 get_submit_target_extra_latest_version() {
     # awk iterates through all the fields in the match to avoid fixed position
     # and it should match with the part which starts with `submit_target_extra_project`+`:`
-    latest_submit_target_extra_version=$(osc dists | grep "$submit_target_extra_project" | awk -v prj="$submit_target_extra_project" '{for (i=1; i<=NF; i++) if ($i ~ "^" prj ":") print $i}' | sort --version-sort | tail -n 1)
+    version_sorted=$(osc dists | grep "$submit_target_extra_project" | awk -v prj="$submit_target_extra_project" '{for (i=1; i<=NF; i++) if ($i ~ "^" prj ":") print $i}' | sort --version-sort | tail -n 2 | tac)
+    local rc=0
```

Comment:
```suggestion
```

Reply:
i think this is needed. I will leave it

Reply:
But what for? It's never read anywhere else